shortstories/views.py

```diff
@@ -1,3 +1,6 @@
-from django.shortcuts import render
+from django.http import HttpResponse
+def index(request):
+    return  HttpResponse('Author,Title,Status,Publication_Date')
+
```

Comment:
The goal is to render shortstories stored in the DB on html, the above code is not correct. I refer you to the tutorial view section.